src/templates/configmap.yaml

```diff
@@ -1,93 +1,16 @@
 apiVersion: v1
 data:
-  credentials: |-
-    {
-       "gcs" : {
-           "gcsCredentialFileName": "gcloud-application-credentials.json"
-       },
-       "s3" : {
-           "s3AccessKeyIDName": "awsAccessKeyID",
-           "s3SecretAccessKeyName": "awsSecretAccessKey"
-       }
-    }
-  explainer: |-
-    {
-        "image" : "seldonio/alibiexplainer:1.1.1-SNAPSHOT"
-    }
-  predictor_servers: |-
-    {
-        "TENSORFLOW_SERVER": {
-          "protocols" : {
-            "tensorflow": {
-              "image": "tensorflow/serving",
-              "defaultImageVersion": "2.1.0"
-              },
-            "seldon": {
-              "image": "seldonio/tfserving-proxy",
-              "defaultImageVersion": "1.6.0"
-              }
-            }
-        },
-        "SKLEARN_SERVER": {
-          "protocols" : {
-            "seldon": {
-              "image": "seldonio/sklearnserver",
-              "defaultImageVersion": "1.6.0"
-              },
-            "kfserving": {
-              "image": "seldonio/mlserver",
-              "defaultImageVersion": "0.1.0"
-              }
-            }
-        },
-        "XGBOOST_SERVER": {
-          "protocols" : {
-            "seldon": {
-              "image": "seldonio/xgboostserver",
-              "defaultImageVersion": "1.6.0"
-              },
-            "kfserving": {
-              "image": "seldonio/mlserver",
-              "defaultImageVersion": "0.1.0"
-              }
-            }
-        },
-        "MLFLOW_SERVER": {
-          "protocols" : {
-            "seldon": {
-              "image": "seldonio/mlflowserver",
-              "defaultImageVersion": "1.6.0"
-              },
-            "kfserving": {
-              "image": "seldonio/mlserver",
-              "defaultImageVersion": "0.1.0"
-              }
-            }
-        },
-        "TRITON_SERVER": {
-          "protocols" : {
-            "kfserving": {
-              "image": "nvcr.io/nvidia/tritonserver",
-              "defaultImageVersion": "20.08-py3"
-              }
-            }
-        }
-    }
-  storageInitializer: |-
-    {
-        "image" : "gcr.io/kfserving/storage-initializer:v0.4.0",
-        "memoryRequest": "100Mi",
-        "memoryLimit": "1Gi",
-        "cpuRequest": "100m",
-        "cpuLimit": "1"
-    }
+  credentials: '{"gcs":{"gcsCredentialFileName":"gcloud-application-credentials.json"},"s3":{"s3AccessKeyIDName":"awsAccessKeyID","s3SecretAccessKeyName":"awsSecretAccessKey"}}'
```

Comment:
Can you run each of these json blobs through an autoformatter? That will prevent horizontal scroll, and also minimize the diff. https://beautifier.io/ is pretty handy if you don't have an autoformatter already.

Reply:
Absolutely, I asked you guys for a formatter, but then forgot to format it. Hehe, will do.

Reply:
Updated it.